docs/user-guide/ze-working-with-data-sets.md

```diff
@@ -0,0 +1,248 @@
+2. Open the profile and data set containing the member you want to delete.
+3. Right-click the member and select **Delete Member**.
+4. Confirm the deletion by selecting **Delete** on the **picker** drop-down menu.
+5. To delete a data set, right-click the data set and select **Delete Data Set**, then confirm the deletion.
+
+   **Note:** You can delete a data set before you delete its members.
+
+   ![Delete](../images/ze/ZE-delete-ds.gif)
+
+## Viewing data set, member attributes
+
+1. Expand **DATA SETS** in the **Side Bar**, and click the **+** icon.
+2. Select the **Search** icon.
+3. In the **picker** field, enter or select a search pattern to filter search results in the **Side Bar**.
+4. Right-click a data set or member and select the **Show Attributes** option.
+
+   This displays the attributes in an **Editor** tab.
+
+    ![View Data Set, Member Attributes](../images/ze/ZE-show-attributes.gif)
+
+## Viewing and accessing multiple profiles simultaneously
+
+1. Expand **DATA SETS** in the **Side Bar**, and click the **+** icon.
+2. Select the profiles from the **picker** drop-down to add them to the **Side Bar**.
+3. Click the **Search** icon for each profile to search and select associated data sets.
+
+   ![Add Profile](../images/ze/ZE-mult-profiles.gif)
+
+## Filtering data set members
+
+Filter partitioned data set members in the **DATA SETS** tree view by **Date Modified** or **User ID**.
+
+### Filtering all data set members under a specific profile
+
+1. In the **DATA SETS** tree, click on the **Filter** icon to the right of a profile, or right-click on a profile and select the **Filter PDS members…** option.
+
+   The filter selection menu appears in the **picker** field.
+2. Select a filter type from the list of available options:
+   - **Date Modified**
+   - **User ID**
+3. Enter a valid value for the selected filter.
+4. Press the `Enter` key to confirm the filter.
+
+   Expanded data sets display a filtered list of members under the selected profile in the **DATA SETS** tree.
+
+   ![Filtering all PDS members under a specific profile](../images/ze/ZE-filtering-profile-PDS-members.gif)
+
+### Filtering members for a single data set
+
+1. In the **DATA SETS** tree, right-click on a data set and select the **Filter PDS members…** option.
+
+   The filter selection menu appears in the **picker** field.
+2. Select a filter type from the list of available options:
+   - **Date Modified**
+   - **User ID**
+3. Enter a valid value for the selected filter.
+4. Press the `Enter` key to confirm the filter. This overrides any *profile* filter preferences that might be in effect for the single data set.
+
+   The selected data set displays a filtered list of members in the **DATA SETS** tree.
+
+   ![Filtering PDS members for a single PDS](../images/ze/ZE-filtering-a-specific-PDS.gif)
+
+## Sorting data set members
+
+Sort partitioned data set members in the **DATA SETS** tree view by member **Name**, **Date Modified**, or **User ID**.
```

Comment:
@traeok -- I noticed that in your explanation you used "PDS" for data sets. For the  most part, I decided to use plain "data sets" b/c we never use "partitioned data sets" in the ZE Sample Uses Cases article, we simply refer to data sets as, well, "data sets." Please let me know if there is a particular reason why we should be using the phrase "partitioned data sets" instead.
CC: @adam-wolfe

Reply:
The sorting and filtering options only apply to members of partitioned data sets - but, I see that "partitioned" is in the description before the steps, so that should be fine 👍

Reply:
thank you -- added "partitioned" back to headers for better clarity